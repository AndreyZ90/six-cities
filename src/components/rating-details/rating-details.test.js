import React from 'react';
import TestRenderer from 'react-test-renderer';

import RatingDetails from '@/components/rating-details/rating-details';

describe(`RatingDetails component snapshot`, () => {
  test(`Should correctly render RatingDetails component`, () => {
    const tree = TestRenderer.create(<RatingDetails rating={4.1} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
