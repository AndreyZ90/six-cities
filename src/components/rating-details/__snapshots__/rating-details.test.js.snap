// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingDetails component snapshot Should correctly render RatingDetails component 1`] = `
<div
  className="property__rating rating"
>
  <div
    className="property__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "80%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  <span
    className="property__rating-value rating__value"
  >
    4.1
  </span>
</div>
`;
